app/models/katello/erratum_bugzilla.rb

```diff
@@ -1,15 +1,4 @@
 #
```

Comment:
Yea, it looks like some of these license headers have an extra commented line at the end.